antarest/study/business/areas/hydro_management.py

```diff
@@ -27,6 +30,23 @@ class HydroManager:
     def __init__(self, command_context: CommandContext) -> None:
         self._command_context = command_context
 
+    def get_all_hydro_properties(self, study: StudyInterface) -> dict[str, HydroProperties]:
+        all_hydro_properties: dict[str, HydroProperties] = {}
+
+        file_study = study.get_files()
+        hydro_management_file_data = HydroManagementFileData(**file_study.tree.get(HYDRO_PATH))
+
+        all_areas = file_study.config.areas
+        for area_id in all_areas:
+            hydro_management = hydro_management_file_data.get_hydro_management(area_id)
+            inflow_structure = self.get_inflow_structure(study, area_id)
+            hydro_properties = HydroProperties.model_validate(
```

Comment:
We can use the constructor instead of the dictionary based form, for better type checking